common/server_room.cpp

```diff
@@ -232,6 +234,18 @@ void Server_Room::say(const QString &userName, const QString &s, bool sendToIsl)
     event.set_name(userName.toStdString());
     event.set_message(s.toStdString());
     sendRoomEvent(prepareRoomEvent(event), sendToIsl);
+
```

Comment:
I mean: if the user set the "roomlist\1\chathistorysize" option to 0 because he doesn't want to retain any log, we don't need to create chatMessage, populate it, add it to chatHistory and the remove it. Just skip lines 238-248